packages/koa-shopify-auth/src/verify-request/verify-request.ts

```diff
@@ -23,6 +22,20 @@ export default function verifyRequest({
 
     if (session && session.accessToken) {
       ctx.cookies.set(TOP_LEVEL_OAUTH_COOKIE_NAME);
+      // we need to check if the accessToken is valid.
```

Comment:
I'd prefer if this comment was a tiny bit more detailed as to why we have to do it